src/main/java/pakiet/Pracownik.java

```diff
@@ -33,4 +33,6 @@ public boolean jestBogaczem() {
     public String duzeImie() {
         return imie.toUpperCase();
     }
+
+    public boolean jestBiedakiem() { return wyplata < 1000; }
```

Comment:
Magic number - please create constant from it.

Reply:
constant created